omnigibson/envs/sb3_vec_env.py

```diff
@@ -0,0 +1,110 @@
+import copy
+import time
+
+import numpy as np
+from stable_baselines3.common.vec_env import DummyVecEnv
+from stable_baselines3.common.vec_env.base_vec_env import VecEnvStepReturn
+from tqdm import trange
+
+import omnigibson as og
+from omnigibson.envs.env_base import Environment
+
+# Keep track of the last used env and what time, to require that others be reset before getting used
+last_stepped_env = None
+last_stepped_time = None
+
+
+class SB3VectorEnvironment(DummyVecEnv):
```

Comment:
Should this inherit from both our VectorEnvironment base class and DummyVecEnv class? It would be nice to have this example to show how to integrate our intended OG vectorized with some external 3rd party requirements (in this case, from SB3). @cgokmen